docs/Getting-Started.md

```diff
@@ -0,0 +1,270 @@
+## Tux Getting Started Guide
+To get started hacking with `Tux`, let's see what some parts of a Todo component might look like inside a To-Do Application built with `Tux`.
+
+## Tux Environment Setup
+Prior to getting started on the Todo application using `Tux` we need setup our working environment. `Tux` is built with CommonJS and thus you will need a compiler such as `Browserify` or `webpack`. In our case, we use `Browserify` for compiling, `Reactify` for compiling JSX, `Envify` for accessing NODE_ENV variables in the browser (great for automatically turning dev tools on and off), and `Watchify` for automatic compiling.
+
+    npm install --save browserify
+    npm install --save envify
+    npm install --save reactify
+    npm install --save watchify
+
+To use these modules in development, we recommend adding the following lines to the `scripts` key of your `package.json` file to set up automatic JSX compiling and configure testing:
+
+    "scripts": {
+      "start": "watchify -d [YOUR MAIN.JS OR SIMILAR FILE] -o [YOUR OUTPUT BUNDLE.JS] -v",
+      "build": "NODE_ENV=production browserify [YOUR MAIN.JS OR SIMILAR FILE] | uglifyjs -cm > [YOUR OUTPUT BUNDLE.JS]"
+    }
+
+Include this `browserify` transform in your `package.json` file as well:
+
+    "browserify": {
+      "transform": [
+        [
+          "reactify",
+          {
+            "es6": true
+          }
+        ],
+        "envify"
+      ]
+    }
+
+The `browserify` transform will cause `watchify` to use `reactify` to compile your JSX and `es6` syntax like the `Spread` syntax, and allow you to use `envify` for node environment variables in the browser.
+
+After adding this code, run
+
+    npm start
+
+during development to automatically compile all JSX and Javascript syntax into one `bundle.js` which you can then directly link to your `index.html` file. Use
+
+    npm run build
+
+to compile a production ready bundle of your code.
+
+## TodoCreateForm.jsx
+Let's start by creating the top level Todo component using `Tux's` `createOwnerClass`. The Owner class is also responsible for the state of our Todo application. In `Tux` you are able to easily connect your Owner components to their stores using the `connectOwnerToStore` method which allows you to designate static properties and methods you want to automatically expose child components on the key of `nearestOwnerProps`. As the top-level Todo component it is also be responsible for exposing its methods and properties its children components. This is accomplished by using the `registerOwnerProps` method, which allows you to specify which properties and methods you want to expose to children components. Note these properties and methods should be static, if you need to pass down mutable properties, use the standard `React` practices to flow down `props`.
```

Comment:
Hmm i think it might be good to cut this down a bit, Since this is a getting started guide people are probably only interested in the code samples anyway. Maybe something like:

Remove also from second sentence.

Shorten everything after second sentence to: The `OwnerClass` possesses tools allowing it to automatically expose static methods and properties to its child components and listen to changes from stores. In this application we will use an `OwnerClass` to manage our todo components, trigger todo actions, and listen to todo store change events.